File: Java/Working with Numbers/50. Count the number of days in a given month of a year.java
import java.util.Scanner;

class Main{
    public static void main(String [] args) {
        Scanner userInput = new Scanner(System.in);
        System.out.println("Enter the Month in number:");
        int month = userInput.nextInt();
        
        System.out.println("Enter the Year:");
        int year = userInput.nextInt();
        
        switch (month) {
            case 1:             //Fall through next statements
            case 3:
            case 5:
            case 7:
            case 8:
            case 10:
            case 12:
                System.out.println("Month "+ month+ " of "+ year+ " has 31 days");
                break;
  
            case 4:
            case 6:
            case 9:
            case 11:
                System.out.println("Month "+ month+ " of "+ year+ " has 30 days");
                break;

            case 2:
                if(( year % 400 == 0 ) || (( year % 100 != 0 ) && ( year % 4 == 0 ))) {
                    System.out.println("Month "+ month+ " of "+ year+ " has 29 days");
                    break;
                }
                else {
                    System.out.println("Month "+ month+ " of "+ year+ " has 28 days");
                    break;
                }
  
            default:
                System.out.println("Invalid Month.");
                break;
        }
        userInput.close();
    }
}

File: Python/02. Working with Numbers/50. Count the number of days in a given month of a year.py
month = int(input("Enter the Month in number:"))
year = int(input("Enter the Year:"))
    
if(( month == 2 ) and (( year % 400 == 0 )  or (( year % 100 != 0 ) and ( year % 4 == 0 )))):
    print("Month", month, "of", year, "has 29 days")

elif( month == 2 ):
    print("Month", month, "of", year, "has 28 days")

elif( month == 1 or month == 3 or month == 5 or month == 7 or month == 8 or month == 10 or month == 12 ):
    print("Month", month, "of", year, "has 31 days")

else:
    print("Month", month, "of", year, "has 30 days")